cabinet/utils.ts

```diff
@@ -123,6 +124,20 @@ export async function reloadCaddy() {
       terminal: true,
     });
 
+    let specifiedEmail = null;
+    try {
+      specifiedEmail = fs.readFileSync(
+        `/home/${domain.username}/.error-page-email`,
+        "utf8",
+      );
+    } catch (err) {
+      console.error("Failed to read specified email:", err);
```

Comment:
This would spam logs a lot for no reason

Reply:
We now only throw errors if the error isn't a "file not found" error.